content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,164 @@
+SCITT supports the ongoing verification of goods and services where the authenticity of entities, evidence, policy, and artifacts can be assured and the actions of entities can be guaranteed to be authorized, non-repudiable, immutable, and auditable.
+
+To assure insights to supply chain artifacts are current, the SCITT APIs provide a correlation of statements for a specific artifact, allowing newer information to be registered for the most up to date insights.
+
+This quickstart will:
+
+1. create, or use an existing a key to sign a collection of statements about an artifact
+1. create and register an SBOM for the artifact
+1. create and register an attestation for the artifact
+1. query a collection of statements about the artifact
+
+## Prerequisites
+
+- [A DataTrails subscription](https://app.datatrails.ai/signup)
+- [DataTrails sample code](#datatrails-sample-code)
+- [SBOM Tool](https://github.com/microsoft/sbom-tool)
+
+### DataTrails Sample Code
+
+Clone the [DataTrails SCITT Examples](https://github.com/datatrails/datatrails-scitt-samples) repository
+
+```bash
+git clone https://github.com/datatrails/datatrails-scitt-samples.git
+
+cd datatrails-scitt-samples
+```
+
+## Create a Signing Key
+
+{{< note >}}
+If you already have a COSE Key, skip ahead to [Register a SBOM for the Artifact](#register-a-sbom-for-the-artifact)
+{{< /note >}}
+
+DataTrails supports multiple signing keys.
+_(TODO: link to supported DataTrails Signing Keys\)_<br>
+For a quickstart, we'll create a testing [COSE Key](https://cose-wg.github.io/cose-spec/#key-structure) which DataTrails will cryptographically validate upon registration
+
+1. Create a local signing key
+
+    ```shell
+    openssl ecparam -name prime256v1 -genkey -out scitt-signing-key.pem
+    ```
+
+## Register a SBOM for the Artifact
+
+When registering statements about an artifact, a common identifier is required to correlate a collection of statements.
+In SCITT, a `CWT_Claims subject` property captures the identifier.
+
+The example generates an [SPDX SBOM](https://spdx.dev/), which generates a [Document Namespace](https://spdx.github.io/spdx-spec/v2.2.2/document-creation-information/#65-spdx-document-namespace-field) field which is used for the `CWT_Claims subject` property.
+
+_\<TODO: Add a doc for creating unique identifiers>_
+
+1. Using the [SBOM Tool](https://github.com/microsoft/sbom-tool), generate an SPDX SBOM for `artifact.js`
+
+    ```bash
+    sbom-tool generate -D true \
+      -b scitt/artifact \
+      -ps scitt-community \
+      -pn scitt-nodejs-example -pv 0.0.0 \
+      -nsb https://scitt.io/examples
+    ```
+
+    [More info for the SPDX sbom-tool](https://github.com/microsoft/sbom-tool/blob/main/docs/sbom-tool-arguments.md)
+1. Capture the Document Namespace for the SCITT `subject`
+
+    ```shell
+    SUBJECT=$(cat scitt/artifact/_manifest/spdx_2.2/manifest.spdx.json | jq -r .documentNamespace)
+    ```
+
+1. Sign and Register the SBOM
```

Comment:
Minor: the step after generating the sbom should be creating the signed statement:

In the root of the scitt samples repo

First we setup a venv:
```
python -m  venv && \
source venv/bin/activate && \
pip install -r requirements.txt
```

Then run the script to create a signed statement:

```
python scitt/create_signed_statement.py --signing-key-file scitt-signing-key.pem \
    --statement-file scitt-statement.json \
    --feed testfeed \
    --issuer testissuer \
```

Finally deactivate the venv:
```
deactivate
```

Reply:
Updated